### DISCUSSION TITLE
مشکل ارور 403 گوگل یا ریکپچا

### DISCUSSION THREAD

**Discussion by abedin70** [Gozargah/Marzban#368] [General]
هنوز که هنوزه خیلی از دوستان رو میبینم که مشکل ارور 403 گوگل یا ریکپچا دارند
( بااینکه خیلی تو یوتیوب ازش ویدیو هست)

اما اگر خیلی خلاصه بخوام بگم برای دوستانی که مشکل این ارور رو دارن:👇

مرحله اول : این اسکریپت زیر رو وارد سرورتون کنید (سرور خارج)

curl -O https://raw.githubusercontent.com/jinwyp/one_click_script/master/install_kernel.sh && chmod +x ./[install_kernel.sh](http://install_kernel.sh/) && ./[install_kernel.sh](http://install_kernel.sh/)

مرحله دوم: سوالاتی که ازتون پرسیده میشه رو دقیقا به همین ترتیب پاسخ بدید:👇
2
12
y
y
1
تموم شد و بعدش مشکل ارور 403 گوگل و ریکپچای سرورتون رفع میشه❤️

نکته: دوستانی که در جریان این اسکریپت نیستن شاید برای اولین بار وارد سرورشون بکنن بترسن که یه وقت به مشکل بخوره سرورشون.
(چون متن های قرمز و رنگی زیادی در طول انجام عملیات داره)
اما نگران نباشید هیچ کاری به اطلاعات شما نداره و شما به ترتیبی که نوشتم پاسخ بدید و مشکلتون حل میشه...

**Comment by mohammadi-hamid** [comment]
بدون مشکل نصب شد اما همچنان خطای ۴۰۳ رو میگیرم

**Reply by abedin70** [reply]
یجای کار ایراد داره
از این استفاده کن
[https://github.com/ErfanNamira/FreeIRAN](https://github.com/ErfanNamira/FreeIRAN)

**Reply by RenasPasha** [reply]
این روش تو لینک پایین رو بری بهتره

https://github.com/Gozargah/Marzban/discussions/734

**Comment by ben-aslan** [comment]
تاثیری تو سرعت داره؟

**Reply by abedin70** [reply]
نه
فکر نکنم
برای پایداری سرعت معمولا bbr میزنن
